packages/react-components/react-skeleton/library/src/components/SkeletonItem/useSkeletonItemStyles.styles.ts

```diff
@@ -45,54 +39,68 @@ const useStyles = makeStyles({
   root: {
     position: 'relative',
     overflow: 'hidden',
-    backgroundSize: '300% 100%',
-    backgroundPositionX: 'center',
-    backgroundPositionY: 'center',
-    backgroundAttachment: 'fixed',
-    animationIterationCount: 'infinite',
-    animationDuration: '3s',
-    animationTimingFunction: 'linear',
-    '@media screen and (prefers-reduced-motion: reduce)': {
-      animationDuration: '0.01ms',
-      animationIterationCount: '1',
+
+    ':after': {
+      content: '""',
+      display: 'block',
+      position: 'absolute',
+      inset: '0',
+      animationIterationCount: 'infinite',
+      animationDuration: '3s',
+      animationTimingFunction: 'ease-in-out',
+      '@media screen and (prefers-reduced-motion: reduce)': {
+        animationDuration: '0.01ms',
+        animationIterationCount: '1',
+      },
     },
   },
   wave: {
-    animationName: skeletonWaveAnimation,
-    backgroundImage: `linear-gradient(
-      to right,
-      ${tokens.colorNeutralStencil1} 0%,
-      ${tokens.colorNeutralStencil2} 50%,
-      ${tokens.colorNeutralStencil1} 100%)`,
-    '@media screen and (forced-colors: active)': {
-      backgroundColor: 'WindowText',
+    backgroundColor: tokens.colorNeutralStencil1,
+
+    ':after': {
+      animationName: skeletonWaveAnimation,
+      backgroundImage: `linear-gradient(
+        to right,
+        ${tokens.colorNeutralStencil1} 0%,
+        ${tokens.colorNeutralStencil2} 50%,
+        ${tokens.colorNeutralStencil1} 100%)`,
+      transform: 'translate(-100%) /* @noflip */',
+
+      '@media screen and (forced-colors: active)': {
+        backgroundColor: 'WindowText',
+      },
     },
   },
   waveRtl: {
-    animationName: skeletonWaveAnimationRTL,
-    backgroundImage: `linear-gradient(
-      to right,
-      ${tokens.colorNeutralStencil1} 0%,
-      ${tokens.colorNeutralStencil2} 50%,
-      ${tokens.colorNeutralStencil1} 100%)`,
-    '@media screen and (forced-colors: active)': {
-      backgroundColor: 'WindowText',
+    ':after': {
+      animationName: skeletonWaveAnimationRTL,
+      transform: 'translate(100%) /* @noflip */',
```

Comment:
`@noflip` seems there to be unnecessary, probably it could be removed at all with `waveRtl`, you can check in [the playground ](https://griffel.js.org/try-it-out/?code=JYWwDg9gTgLgBAbziAhgawKYGUYE8A2GAznAL5wBmUEIcA5AAIDmUwFFG%2BA9AMbQZ0A3AChhGAB6RYcACYYKKAK754qTDgLEAFAmFw41CDABciPfrgoAdqBQxgEKwDkUIDKd0W4MCB-P6YKGsiCmgQUzpA4Pw7DC0AWgBGAAZkgFIASjoAGnNSXK99Ujzc0gyRIA)